src/main/java/com/autotune/analyzer/recommendations/utils/RecommendationUtils.java

```diff
@@ -194,7 +194,7 @@ public static void markAcceleratorDeviceStatusToContainer(ContainerData containe
         if (null == resultArray || resultArray.isEmpty()) {
             // Need to alert that container max duration is not detected
             // Ignoring it here, as we take care of it at generate recommendations
```

Comment:
looks like we need to update the comment here

Reply:
I feel we need that comment as these functions `markAcceleratorPartitionDeviceStatusToContainer` & `markAcceleratorDeviceStatusToContainer` try to extract the accelerator details from prometheus and while checking the code we might add alerts (In future) thinking that it's missing here.

This comment helps in understanding that it's not required to alert the user twice about the same thing. Please correct me if I'm wrong.